.github/actions/prepare-install/action.yml

```diff
@@ -55,9 +55,16 @@ runs:
         restore-keys: |
           yarn-download-cache-
 
+    - name: Check if ast-spec files changed
+      id: changed-files
+      uses: tj-actions/changed-files@v41
+      with:
+        files: packages/ast-spec/src/**
```

Comment:
Can you revert this or move this to a separate PR/issue if it's necessary?

Reply:
@Josh-Cena 

>  You don't need to worry about CI failures. As long as your code is valid, the burden of fixing the CI is on the team.

Thanks!!

> Can you revert this or move this to a separate PR/issue if it's necessary?

Undo file changes in github actions.

https://github.com/typescript-eslint/typescript-eslint/pull/9632/commits/a8995448520ef510b896adbd21e3c8a6c96eefcc